### PR TITLE
Update `OWC 2023` for the RO32 stage

### PR DESCRIPTION
**Pull request by LeoFLT**
Outdating the korean translation since there's quite a bit of added text that would need a review.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/OWC/2023/en.md [hunk 1]
```diff
@@ -144,12 +198,78 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/82e
...
   3. [Sable Hills - Not Falling (squirrelpascals) \[Vid\]](https://osu.ppy.sh/beatmapsets/2075253#osu/4344422)
 
+## Results
+
+### Qualifiers
+
+The final standings for the Qualifier stage can be found in the following [spreadsheet](https://docs.google.com/spreadsheets/d/1iih6qvAYpCakmdZcnFoB_XGxBm4WDBYNNeo-81PcKGc?rm=minimal).
+
+| Seed | Country | %MAX sum[^qualifiers-seeding] | avg. score[^qualifiers-tiebreaker] |
+| :-: | :-- | :-- | :-- |
```
Comment (cl8n) [inline wiki/Tournaments/OWC/2023/en.md:208]: for readability of this table--
- right align the last two columns
- *don't* left-pad numbers (average scores of Croatia and India)
- right-pad numbers to use the same precision (e.g. poland %max should read `10.330`)

Comment (LeoFLT) [reply wiki/Tournaments/OWC/2023/en.md]: 🤦 I forgot to apply the correct formatting to the last two tables. Applied all suggested fixes.

File: wiki/Tournaments/OWC/2023/en.md [hunk 2]
```diff
@@ -144,12 +198,78 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/82e
...
+| #56 | ::{ flag=HR }:: Croatia | 2.892 | 0,969,311 |
+| #57 | ::{ flag=IN }:: India | 2.69 | 0,892,297 |
+
 ## Ruleset
 
 ### Tournament rules
 
 1. The osu! World Cup is a country-based 4 versus 4 double-elimination team tournament, played on the osu! game mode.
-   - Players from countries or territories (as seen on their profile flag) that are under a protectorate or sovereign state may choose to play for that country **if** the country listed on their profile is unable to field enough players to form a valid team. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of the criteria adopted.
+   - Players from countries or territories (as seen on their profile flag) that are under a protectorate or sovereign state may choose to play for that country **if** the country listed on their profile is unable to field enough players to form a valid team. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of the criteria adopted. Players that have participated under another country previously may continue to do so.
```
Comment (cl8n) [inline wiki/Tournaments/OWC/2023/en.md:272]: just want to clarify, does this added part apply to *all players* or just the players being talked about on this line so far?

Comment (cl8n) [reply wiki/Tournaments/OWC/2023/en.md:272]: actually the wording is fairly clear rereading it again, nvm.

**Review by cl8n — APPROVED** [review]

**Pull request merged**